include/PR/os_cont.h

```diff
@@ -161,6 +161,12 @@ typedef struct {
 #define CONT_ERR_VOICE_WORD        14
 #define CONT_ERR_VOICE_NO_RESPONSE 15
 
+// Controller mask values
+#define CONT_P1 0x01
+#define CONT_P2 0x02
+#define CONT_P3 0x04
+#define CONT_P4 0x08
```

Comment:
```suggestion
#define CONT_P1 (1 << 0)
#define CONT_P2 (1 << 1)
#define CONT_P3 (1 << 2)
#define CONT_P4 (1 << 3)
```
Let's do the funny bit definitions to make it obvious these are single-bit flags